update/starter/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/temporalio/samples-go/update"
+	"go.temporal.io/sdk/client"
+)
+
+func main() {
+	c, err := client.Dial(client.Options{})
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+
+	workflowOptions := client.StartWorkflowOptions{
+		ID:        "update-workflow-ID",
+		TaskQueue: "update",
+	}
+
+	we, err := c.ExecuteWorkflow(context.Background(), workflowOptions, update.Counter)
+	if err != nil {
+		log.Fatalln("Unable to execute workflow", err)
+	}
+
+	log.Println("Started workflow", "WorkflowID", we.GetID(), "RunID", we.GetRunID())
+
+	mult := 1
```

Comment:
Might as well throw a comment above this saying what it's doing with these values (can of course be small, because README and workflow itself kinda show what's happening)

Reply:
Added on the `*= -1` line.